ci/test-flow-visibility.sh

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 Antrea Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+CLICKHOUSE_URL="tcp://clickhouse-clickhouse.flow-visibility:9000"
+FAILURE=false
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+GIT_CHECKOUT_DIR=${THIS_DIR}/..
+
+_usage="Usage: $0 [--kubeconfig <KubeconfigSavePath>]
+
+Setup Grafana, ClickHouse and Antrea Flow Aggregator to validate flow-aggregator.yml and flow-visibility.yml.
+
+    --kubeconfig    Path to kubeconfig.
+
+"
+
+echoerr() {
+  >&2 echo "$@"
+}
+
+print_usage() {
+  echoerr "$_usage"
+}
+
+
+while [[ $# -gt 0 ]]
+do
+key="$1"
+case $key in
+    --kubeconfig)
+    export KUBECONFIG="$2"
+    echo "kube"
+    shift 2
+    ;;
+    -h|--help)
+    print_usage
+    exit 0
+    ;;
+    *)
+    echoerr "Unknown option $1"
+    exit 1
+    ;;
+esac
+done
+
+start_antrea() {
+  echo "=== Starting Antrea ==="
+  kubectl apply -f ${GIT_CHECKOUT_DIR}/build/yamls/antrea.yml
+  kubectl rollout status --timeout=5m deployment/coredns -n kube-system
+  kubectl rollout status --timeout=5m deployment.apps/antrea-controller -n kube-system
+  kubectl rollout status --timeout=5m daemonset/antrea-agent -n kube-system
+}
+
+setup_flow_visibility() {
+  echo "=== Starting Flow Visibility ==="
+  # install ClickHouse operator
+  kubectl apply -f https://raw.githubusercontent.com/Altinity/clickhouse-operator/master/deploy/operator/clickhouse-operator-install-bundle.yaml
```

Comment:
I  think it's a good idea to test master branch. This will allow us to catch early if any change in the clickhouse operator might break Antrea flow visibility.

In case we need to test a specific clickhouse version, is there a way to parameterize this script in a way that it could pull a specific operator version?

Reply:
Sure, we could add a parameter to test a specific version of the Clickhouse operator. Just wondering when we would like to test a specific version? Since in the flow-visibility doc, we will always pull the yaml file from master branch: https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md#deployment-steps-1

Reply:
Update: 
Flow Aggregator e2e test with Clickhouse has been removed in the latest commits from #3196. So there may be issues testing the yml from the master branch here, because we may not have the latest version of `altinity/clickhouse-operator`, `altinity/metrics-exporter` images. Similar problems apply to other external images:`grafana/grafana` and `yandex/clickhouse-server` too. 
Considering updating our `clickhouse-operator-install-bundle.yaml` to specific version in doc and adding a workflow to push these external images to Antrea docker hub, may I also hear opinions from maintainers @antoninbas @jianjuns @tnqn 
regarding this? Thanks!

Reply:
@antoninbas what's your feedback on @dreamtalen's plan?

Reply:
Finished switching to Antrea docker hub for external images. @salv-orlando

Reply:
I think I'm ok with the approach, which is consistent to what we have done for other images. Maybe all of these mirrored images could be prefixed with `flow-visibility-` though, so that they are "grouped" together and to be consistent with the `flow-visibility-clickhouse-monitor` image. The `metrics-exporter` image in particular has a very generic name right now.

Reply:
Thanks Antonin for the reviews! Sounds good to me, added `flow-visibility-` prefix to all related images.